fetcher.go

```diff
@@ -86,6 +87,11 @@ func (p *pool) doFetch(ctx context.Context, from string, c cid.Cid, attempt int)
 
 // TODO Refactor to use a metrics collector that separates the collection of metrics from the actual fetching
 func (p *pool) fetchResource(ctx context.Context, from string, resource string, mime string, attempt int, cb DataCallback) (rm tieredhashing.ResponseMetrics, err error) {
+	isRange := "no"
+	if strings.Contains(resource, "bytes") {
```

Comment:
* should we parse the URL and make sure there's a query parameter with the name?
* the name should be `entity-bytes` not bytes, right?